aries-site/src/data/wcag/components.json

```
@@ -862,5 +862,519 @@
+      },
+      {
+        "rule": "1.4.9",
+        "label": "Images of Text (No Exception)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.10",
+        "label": "Reflow",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.11",
+        "label": "Non-text Contrast",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.13",
+        "label": "Content on Hover or Focus",
+        "status": "failed",
+        "notes": "Failing because when the file name truncates there isn't a way for keyboard users to access the tooltip"
+      },
+      {
+        "rule": "2.1.1",
+        "label": "Keyboard",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.1.2",
+        "label": "No Keyboard Trap",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.1.3",
+        "label": "Keyboard (No Exception)",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.1",
+        "label": "Timing Adjustable",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.2",
+        "label": "Pause, Stop, Hide",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.3",
+        "label": "No Timing",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.4",
+        "label": "Interruptions",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.5",
+        "label": "Re-authenticating",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.6",
+        "label": "Timeouts",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.1",
+        "label": "Three Flashes or Below Threshold",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.2",
+        "label": "Three Flashes",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.3",
+        "label": "Animation from Interactions",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.1",
+        "label": "Bypass Blocks",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.2",
+        "label": "Page Titled",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.3",
+        "label": "Focus Order",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.4",
+        "label": "Link Purpose (In Context)",
+        "status": "passed",
```

```suggestion
        "status": "not-applicable",
```
We don't use links in this component

---

Accepted change